elao.app/.manala/Makefile.tmpl

```diff
@@ -140,6 +140,31 @@ deploy{{ include "release_target" $release }}:
 
 {{ end -}}
 
+
+########
+# Audit #
+########
+
+HELP += $(call help_section, Audit)
+
+ifneq ($(wildcard composer.lock),)
+HELP += $(call help,audit.composer, Checks for known security issues with composer.)
+audit.composer:
+	symfony check:security
+endif
+
+ifneq ($(wildcard package-lock.json),)
+HELP += $(call help,audit.npm, Checks for known security issues with npm.)
+audit.npm:
+	npm audit ; RC=$${?} ; [ $${RC} -gt 2 ] && exit $${RC} || exit 0
+endif
+
+ifneq ($(wildcard yarn.lock),)
+HELP += $(call help,audit.yarn, Checks for known security issues with yarn.)
+audit.yarn:
+	yarn audit ; RC=$${?} ; [ $${RC} -gt 2 ] && exit $${RC} || exit 0
```

Comment:
I don't know if npm & yarn use the same convention for the exit code

> The command will exit with a non-0 exit code if there are issues of any severity found. The exit code will be a mask of the severities.
> 
> 1 for INFO
> 2 for LOW
> 4 for MODERATE
> 8 for HIGH
> 16 for CRITICAL
> For example, if only INFO and MODERATE vulnerabilities were found, then the exit code will be 1 + 4 = 5

Reply:
Pour NPM, a priori c'est `npm audit --audit-level [low|moderate]`